Xamarin.Forms.Platform.iOS/CollectionView/CollectionViewController.cs

```diff
@@ -179,5 +185,86 @@ void RegisterCells()
 				HorizontalTemplatedCell.ReuseId);
 			CollectionView.RegisterClassForCell(typeof(VerticalTemplatedCell), VerticalTemplatedCell.ReuseId);
 		}
+
+		void UpdateEmptyView(bool isEmpty)
+		{
+			if (isEmpty)
+			{
+				// Is EmptyView set on the ItemsView?
+				var emptyView = _itemsView?.EmptyView;
+
+				if (emptyView == null)
```

Comment:
The name of the variable is confusing, because below is used exactly the same but with the underscore. Is it possible to remove `emptyView` variable here? `if (_itemsView?.EmptyView == null)`

Reply:
Renamed the cached variable to _emptyUIView to make it more explicit. Also, this made me realize I had forgotten to allow for the EmptyView/EmptyViewTemplate to change after the firs time it was displayed - that should be fixed now.